src/Database/PG/Query/Pool.hs

```diff
@@ -229,18 +222,20 @@ mkConnExHandler :: (MonadError e m)
                 -> (PGConnErr -> m a)
 mkConnExHandler _ ef = throwError . ef
 
+-- | Run a command on the given pool wrapped in a transaction.
 runTx :: ( MonadIO m
          , MonadBaseControl IO m
          , FromPGTxErr e
          , FromPGConnErr e
          )
       => PGPool
       -> TxMode
-      -> TxET e m a
+      -> TxET e m a -- ^ the command to run
```

Comment:
`TxET` (and most of the contents of the `Transaction` module are woefully misnamed: These aren't transactions.